### PR TITLE
Add Odos on Arbitrum to dex_aggregator

### PR DESCRIPTION
**Pull request by lequangphu**
### Update!
Please build spells in the proper [subproject](../dbt_subprojects/) directory. For more information, please see the main [readme](../README.md), which also links to a GH discussion with the option to ask questions.

### Contribution type
Please check the type of contribution this pull request is for:

- [ ] New spell(s)
- [x] Adding to existing spell lineage
- [ ] Bug fix

**Note:** You can safely discard any section below which doesn't apply based on selection above

---

### For new spell(s)
If you are building new spell(s), please provide the following information:

- **Spell name(s):** *schema.table_name*
- **Description:** [Detailed description of the new spell(s) and their purpose]
- **Who are the new spell(s) for?** [Internal team or general community]
- **How will the new spell(s) be used downstream?** [Description of downstream usage]
- **Implementation details:** [Information on how the spell(s) are implemented]
- **Test instructions:** [How to test the new spell(s)]
- **Related issue(s):** [Link to related issues, if any]

---

### For adding to existing spell lineage
If you are adding to an existing spell lineage, please provide the following information:

- **Description:** [Description of the changes made]
Add trades of Odos on Arbitrum to dex_aggregator based on existing spell of Odos on Optimism.
---

### For bug fixes
If you are fixing a bug, please provide the following information:

- **Description:** [Description of the bug fix]
- **Steps to reproduce:** [How to reproduce the bug]
- **Implementation details:** [Information on how the bug was fixed]
- **Test instructions:** [How to test the fix]
- **Related issue(s):** [Link to related issues, if any]

---

### Additional information
Please provide any additional information that might help us review your pull request:

- [Any additional information]

---

Thank you for your contribution!

### CODE REVIEW

File: dbt_subprojects/dex/models/_projects/odos/arbitrum/_schema.yml [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+            - blockchain
+            - project
+            - version
+            - tx_hash
+            - evt_index 
+            - trace_address
+      - check_dex_aggregator_seed:
+          blockchain: arbitrum
+          project: odos
+          version: 1
```
[the remark this answers is not in the document]
Comment (lequangphu) [reply dbt_subprojects/dex/models/_projects/odos/arbitrum/_schema.yml:27]: what is the path where i place the seed, the readme file of seed is outdated.

[the remark this answers is not in the document]
Comment (Hosuke) [reply dbt_subprojects/dex/models/_projects/odos/arbitrum/_schema.yml:27]: You can add the seed records in this file:
https://github.com/duneanalytics/spellbook/blob/main/dbt_subprojects/dex/seeds/aggregator_trades/dex_aggregator_seed.csv

[the remark this answers is not in the document]
Comment (Hosuke) [reply dbt_subprojects/dex/models/_projects/odos/arbitrum/_schema.yml:27]: Updated readme here:
https://github.com/duneanalytics/spellbook/blob/befbddab4b46f91c9eb2919a33debab13d1a3b24/dbt_subprojects/dex/models/trades/readme.md

[the remark this answers is not in the document]
Comment (lequangphu) [reply dbt_subprojects/dex/models/_projects/odos/arbitrum/_schema.yml:27]: I added seed records, pls check, thanks.

**Review by Hosuke — APPROVED** [review]
LGTM.✅
Thank you @lequangphu 

[Sample odos_arbitrum_trades](https://dune.com/queries/4058524)

**Pull request merged**